.gitignore

```diff
@@ -51,3 +51,4 @@ flycheck-*
 
 ### vim
 .ropeproject/
+
```

Comment:
Please drop that line as well so that there is no change in this file for your PR.